### ISSUE TITLE
Cannot get profile status by profile ID (CLI flag is missing)

### ISSUE THREAD

**Issue by rdimitrov** [mindersec/minder#4999]
The issue is that we cannot get the status of a profile by its ID because the CLI flag is missing.

**Comment by navnitms** [comment]
I would like to work on this issue

**Comment by rdimitrov** [comment]
hey, @navnitms, of course, I've assigned it to you 👍 Thanks! 💯 🚀

**Comment by navnitms** [comment]
@rdimitrov Just to clarify, am I correct in understanding that the requirement is to add a profileId flag to the "Get Profile Status" command?

**Comment by rdimitrov** [comment]
Yes, one part of the task is to have something similar to the `-i/--id` flag we have for the `profile get` command. 

The other one is to add a new `GetProfileStatusById` handler similar to `GetProfileStatusByName` so you can tie all of this together 👍

**Comment by navnitms** [comment]
I've started working on this issue

**Comment by JAORMX** [comment]
Thank you!

**Comment by rdimitrov** [comment]
Closing as completed 👍 Thank you @navnitms 💯
